src/components/app/details/cicdHistory/History.components.tsx

```diff
@@ -0,0 +1,127 @@
+import React, { useEffect } from 'react'
+import { not, useKeyDown } from '../../../common'
+import { useLocation } from 'react-router'
+import Tippy from '@tippyjs/react'
+import { ReactComponent as ZoomIn } from '../../../../assets/icons/ic-fullscreen.svg'
+import { ReactComponent as ZoomOut } from '../../../../assets/icons/ic-exit-fullscreen.svg'
+import { ReactComponent as DropDownIcon } from '../../../../assets/icons/ic-chevron-down.svg'
+import { ReactComponent as OpenInNew } from '../../../../assets/icons/ic-open-in-new.svg'
+import AppNotDeployed from '../../../../assets/img/app-not-deployed.png'
+import { CiMaterial, EmptyViewType, GitChangesType, GitTriggers, LogResizeButtonType, ScrollerType } from './types'
+import GitCommitInfoGeneric from '../../../common/GitCommitInfoGeneric'
+import EmptyState from '../../../EmptyState/EmptyState'
+import { NavLink } from 'react-router-dom'
+
+export const LogResizeButton = ({ fullScreenView, setFullScreenView }: LogResizeButtonType): JSX.Element => {
+    const { pathname } = useLocation()
+
+    const keys = useKeyDown()
+
+    useEffect(() => {
+        if (!pathname.includes('/logs')) return
+        switch (keys.join('')) {
+            case 'f':
+                setFullScreenView(not)
+                break
+            case 'Escape':
+                setFullScreenView(false)
+                break
+        }
+    }, [keys])
+
+    const hideFullScreen = (): void => {
+        setFullScreenView(false)
+    }
+
+    const showFullScreen = (): void => {
+        setFullScreenView(true)
+    }
+    return pathname.includes('/logs') ? (
+        <Tippy
+            placement="top"
+            arrow={false}
+            className="default-tt"
+            content={fullScreenView ? 'Exit fullscreen (f)' : 'Enter fullscreen (f)'}
+        >
+            {fullScreenView ? (
+                <ZoomOut className="zoom zoom--out pointer" onClick={hideFullScreen} />
+            ) : (
+                <ZoomIn className="zoom zoom--in pointer" onClick={showFullScreen} />
+            )}
+        </Tippy>
+    ) : null
+}
+
+export const Scroller = ({ scrollToTop, scrollToBottom, style }: ScrollerType): JSX.Element => {
+    return (
+        <div
+            style={{ ...style, display: 'flex', flexDirection: 'column', justifyContent: 'top' }}
+            className="dc__element-scroller"
```

Comment:
can we use classes like `flex column top` instead of inline style?

Reply:
Done